src/Ssh/Ssh/Common/SshBaseCmdlet.cs

```diff
@@ -622,6 +575,36 @@ protected internal bool IsArc()
             return false;
         }
 
+        /*
+         * Method for checking if Az.Ssh.ArcProxy module is installed
+         */
+        protected internal string FindProxyModuleInstalled()
+        {
+            var results = InvokeCommand.InvokeScript(
+                script: "(Get-module -ListAvailable -Name Az.Ssh.ArcProxy).Path");
+
+            if (results.Count == 0)
```

Comment:
We don't need this check here.  If `results.Count` is zero, then we will fall through the foreach loop below.